src/pages/VaultsPage/VaultRow.tsx

```diff
@@ -66,11 +68,16 @@ export function VaultRow({ vault, css = {} }: { vault: Vault; css?: CSS }) {
         onUpdateBalance={updateBalance}
       />
       <Box
-        css={{ display: 'flex', alignItems: 'center', gap: '$md', mb: '$md' }}
+        css={{ display: 'flex', alignItems: 'center', gap: '$md', mb: '$xs' }}
       >
         <Text h3 css={{ flexGrow: 1 }}>
           {vault.symbol || '...'} CoVault
+          <VaultExternalLink
```

Comment:
in order for this to have the correct spacing next to the rest of the text, it should be set to `display: inline` (or inline-block) rather than using a fixed left margin

Reply:
tried and seem to only work when i used `&nbsp;`